src/components/FilterPanel/FilterPanel.test.tsx

```diff
@@ -0,0 +1,42 @@
+import { render, screen, fireEvent } from '@testing-library/react';
+import { describe, it, expect, vi } from 'vitest';
+
+import { FilterPanel } from './FilterPanel';
+
+describe('FilterPanel', () => {
+  it('resalta el filtro seleccionado', () => {
```

Comment:
Test description is written in Spanish. Project requires English as the standard language for code, tests and docs.

**Sugerencia:**
```typescript
Translate the test description to English. Example: it('highlights the selected filter', () => { ... })
```